src/crewai/utilities/events/base_events.py

```diff
@@ -3,12 +3,26 @@
 
 from pydantic import BaseModel, Field
 
+from crewai.flow.state_utils import to_serializable
 
-class CrewEvent(BaseModel):
-    """Base class for all crew events"""
+
+class BaseEvent(BaseModel):
```

Comment:
Every event from flows to crews were based in this class, so I renamed it to `BaseEvent`.